interfaces/MCube-Accel-MC36xx/src/platform/user_periph_setup.c

```diff
@@ -82,7 +82,7 @@ void periph_init(void)
     SetBits16(CLK_16M_REG, XTAL16_BIAS_SH_ENABLE, 1);
 
     //rom patch
-    patch_func();
+//    patch_func();
```

Comment:
Why? If this is not necessary, please remove.

Reply:
solved